faqs/README.md

```diff
@@ -144,10 +144,12 @@ The following prices are exclusive of any local taxes and shipping/handling fees
 
 ### What do I get when I buy one?
 
-You get the Raspberry Pi board itself. The [official power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) and an SD card are not included, but they can be purchased at the same time from most places that sell the Raspberry Pi. You can also purchase pre-loaded SD cards; we recommend buying these from us or our licensed distributors rather than from third parties, as our software is updated regularly and cards sold by third parties can quickly become outdated.
+Raspberry Pi devices are available as just the board, or as part of a kit. When buying just the device, you get the Raspberry Pi board itself. The [official power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) and an SD card are not included, but they can be purchased at the same time from most places that sell the Raspberry Pi. You can also purchase pre-loaded SD cards; we recommend buying these from us or our licensed distributors rather than from third parties, as our software is updated regularly and cards sold by third parties can quickly become outdated.
 
 An official case for the Raspberry Pi is available separately [from our distributors](https://www.raspberrypi.org/products/). There are also lots of home-brew case discussions on the forum, as well as several third-party cases available. There are also loads of awesome [3D-printable cases](https://www.raspberrypi.org/blog/3d-printed-raspberry-pi-cases/) that community members have designed.
 
+Raspberry Pi (and other sellers) also sell our devices as part of kits, where you get everything you need to set up your Raspberry Pi and get it going. These kits usually include the [Raspberry Pi](https://www.raspberrypi.org/products/raspberry-pi-4-desktop-kit/?resellerType=home) itself, a case, keyboard, mouse, power supply and SD card. In the case of the [Raspberry Pi 400](https://www.raspberrypi.org/products/raspberry-pi-400/?resellerType=home), the device itself is the case and keyboard, but the kit includes a mouse, power supply, SD card and a book, The Official Raspberry Pi Beginners Guide.
```

Comment:
Can probably remove the `?resellerType=home` from the URLs.

The Pi 4 Desktop kit also includes a book?

Might be worth mentioning that an HDMI cable is included too? (i.e. the kit is basically "everything you need except for the TV / monitor")

It might be worth trying to differentiate between the "official kits", and the many 3rd-party kits that resellers put together?

Reply:
Ah, forgot the Pi4 kit had a book as well. Will also add cables, forgot that!

Not sure how to deal with third party kits except to say they exist.

Reply:
> Not sure how to deal with third party kits except to say they exist.

Perhaps the Communications or Commercial teams can suggest suitable wording ?

Reply:
Just checking.